### PR TITLE
Syncing Russian README with the original

### PR DESCRIPTION
**Pull request by shurup**
The Russian README file is outdated, e.g. it lacks the up-to-date recommendation & instructions for running the website using a container. This PR brings the file in sync. It also improves a few existing phrasings and reorders the list of READMEs on different languages according to the Russian alphabet.

### CODE REVIEW

**Comment by shurup** [comment]
I signed it

File: README-ru.md [hunk 1]
```diff
@@ -2,38 +2,117 @@
...
 cd website
-hugo server --buildFuture
 ```
 
-Эта команда запустит сервер Hugo на порту 1313. Откройте браузер и перейдите по ссылке http://localhost:1313, чтобы открыть сайт. Если вы отредактируете исходные файлы сайта, Hugo автоматически применит изменения и обновит страницу в браузере.
+Сайт Kubernetes использует [тему для Hugo под названием Docsy](https://github.com/google/docsy). Даже если вы планируете запускать сайт в контейнере, мы настоятельно рекомендуем загрузить соответствующий сабмодуль и другие зависимости для разработки, выполнив следующую команду:
 
-## Сообщество, обсуждение, вклад и поддержка
+```
+# pull in the Docsy submodule
```
Comment (sftim) [inline README-ru.md:30]: If you want to, you can localize this comment

File: README-ru.md [hunk 2]
```diff
@@ -2,38 +2,117 @@
...
+Попробуйте проверить текущий лимит для открытых файлов:
+
+`launchctl limit maxfiles`
+
+Затем выполните следующие команды (они взяты и адаптированы из https://gist.github.com/tombigel/d503800a282fcadbee14b537735d202c):
+
+```shell
+#!/bin/sh
+
+# These are the original gist links, linking to my gists now.
```
Comment (sftim) [inline README-ru.md:82]: If you want to localize this comment, it's OK

File: README-ru.md [hunk 3]
```diff
@@ -42,21 +121,22 @@ hugo server --buildFuture
...
+| [Индонезийский](README-id.md) | [Португальский](README-pt.md) |
+| [Испанский](README-es.md)     | [Украинский](README-uk.md)    |
+| [Итальянский](README-it.md)   | [Французский](README-fr.md)   |
+| [Китайский](README-zh.md)     | [Хинди](README-hi.md)         |
+| [Корейский](README-ko.md)     | [Японский](README-ja.md)      |
 
-### Кодекс поведения
+# Кодекс поведения
 
 Участие в сообществе Kubernetes регулируется [кодексом поведения CNCF](https://github.com/cncf/foundation/blob/master/code-of-conduct.md).
```
Comment (sftim) [inline README-ru.md:138]: Optionally: link to https://github.com/cncf/foundation/blob/master/code-of-conduct-languages/ru.md instead.

**Review by sftim — COMMENTED** [review]
Hi. A few optional, and minor, suggestions.

(I don't speak Russian well enough to give a full review).

File: README-ru.md (hunk 3 above)
Comment (shurup) [reply README-ru.md]: All done in a784cb42f855660d79a6d058c1cfb93021d1eb11.

**Review by Potapy4 — APPROVED** [review]
/lgtm
/approve

**Comment by sftim** [comment]
/approve
echoing https://github.com/kubernetes/website/pull/25808#pullrequestreview-559064777

**Pull request merged**